applications/directory/src/pages/profile/[token]/edit.js

```diff
@@ -51,19 +57,51 @@ const ProfileReviewEditPage = () => {
             <h1>Edit artist profile</h1>
           </Box>
 
-          <Box
-            sx={{
-              display: 'flex',
-              justifyContent: 'space-between',
-              alignItems: 'flex-start'
-            }}
-          >
-            <CreateProfileForm />
-          </Box>
+          <Card elevation={6}>
+            <Loader
+              isLoading={
+                categoriesLoading ||
+                tagsLoading ||
+                skillsLoading ||
+                artistLoading
+              }
+              isError={
+                categoriesError || tagsError || skillsError || artistError
+              }
+              loadingComponent={() => (
+                <LinearProgress color="primary"></LinearProgress>
+              )}
+              errorComponent={() => (
+                <Alert
+                  severity="error"
+                  sx={{
+                    fontSize: '1.2rem'
+                  }}
+                  elevation={4}
+                >
+                  An unexpected error occurred. Please try again shortly.
+                </Alert>
+              )}
+            >
+              <CreateProfileForm
+                skills={skills}
+                tags={tags}
+                categories={categories}
+                artist={artist}
+              />
+            </Loader>
+          </Card>
         </Layout.Root>
       </Layout>
     </>
   );
 };
 
+export async function getServerSideProps(context) {
```

Comment:
Ah ha! Using this instead of router? Works well?

Reply:
It works better for sure! Thank you!